### ISSUE TITLE
Add `Tx.transfer` factory method with coin selection

### ISSUE THREAD

**Issue by troggy** [leapdao/leap-core#34]
To replace this boilerplate:

```
const inputs = helpers.calcInputs(
  unspent,
  this.account.address,
  amount as number,
  this.color
);
const outputs = helpers.calcOutputs(
  unspent,
  inputs,
  this.account.address,
  to,
  amount as number,
  this.color
);
return Tx.transfer(inputs, outputs);
```

with something like `Tx.transfer(from, to, value, color)`

**Comment by eezcjkr** [comment]
This is impossible.

**Comment by eezcjkr** [comment]
Ok it is semi-possible

**Comment by troggy** [comment]
closed via #43
